Objects/longobject.c

```diff
@@ -5542,12 +5543,14 @@ Return an array of bytes representing an integer.
 static PyObject *
 int_to_bytes_impl(PyObject *self, Py_ssize_t length, PyObject *byteorder,
                   int is_signed)
-/*[clinic end generated code: output=89c801df114050a3 input=ddac63f4c7bf414c]*/
+/*[clinic end generated code: output=89c801df114050a3 input=5af4e767317e06a4]*/
 {
     int little_endian;
     PyObject *bytes;
 
-    if (_PyUnicode_EqualToASCIIId(byteorder, &PyId_little))
+    if (byteorder == NULL)
+        little_endian = 0;
+    else if (_PyUnicode_EqualToASCIIId(byteorder, &PyId_little))
         little_endian = 1;
     else if (_PyUnicode_EqualToASCIIId(byteorder, &PyId_big))
         little_endian = 0;
```

Comment:
I really like the order of the logic here (and in `from_bytes`) now that we have this change. Using the default (`"big"`) is super-fast, manually passing the non-default (`"little"`) is a bit slower, and manually passing the default (`"big"`) is slowest. That seems right to me.

Reply:
Indeed!  I like how that worked out too!